packages/-ember-data/tests/unit/adapters/rest-adapter/fetch-options-test.js

```diff
@@ -146,4 +146,32 @@ module('unit/adapters/rest-adapter/fetch-options', function(hooks) {
     const postOptions = fetchOptions(postData);
     assert.equal(postOptions.body, stringBody, "'options.body' is the String passed in");
   });
+
+  test("fetchOptions sets credentials when 'credentials' is empty", function(assert) {
+    assert.expect(1);
```

Comment:
Since we have sync path to asserting, we could delete the expect if you wanted

Reply:
@snewcomer Okay! What should I do?

Reply:
@kookyungmin Nope!  Other than bringing in the latest of the main branch to get these tests passing.

Reply:
Oh I passed all the tests!